src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1394,13 +1374,8 @@ static void query_for_backends_locked(grpc_exec_ctx *exec_ctx,
   op->flags = 0;
   op->reserved = NULL;
   op++;
-  /* take a weak ref (won't prevent calling of \a glb_shutdown if the strong ref
-   * count goes to zero) to be unref'd in lb_on_sent_initial_request_locked() */
-  GRPC_LB_POLICY_WEAK_REF(&glb_policy->base,
-                          "lb_on_sent_initial_request_locked");
-  call_error = grpc_call_start_batch_and_execute(
-      exec_ctx, glb_policy->lb_call, ops, (size_t)(op - ops),
-      &glb_policy->lb_on_sent_initial_request);
+  call_error = grpc_call_start_batch_and_execute(exec_ctx, glb_policy->lb_call,
```

Comment:
I forget, and of course we don't have docs in the .h, but `grpc_call_start_batch_and_execute` makes a copy/takes a ref of its inputs, correct? Perusing the implementation indicates so, but in case I've missed something.

Reply:
The ownership semantics for most ops are documented in grpc_types.h.  For example, the send_message semantics are documented here:

https://github.com/grpc/grpc/blob/master/include/grpc/impl/codegen/grpc_types.h#L518

These docs could certainly be better, and we can improve them separately.

In any case, this PR does not change anything related to ownership of the arguments being passed into `grpc_call_start_batch_and_execute()`, so regardless of the behavior, it's not affected by this PR.